src/future/index.js

```diff
@@ -35,7 +34,7 @@ export function FormContainer({
         onChange={onChange}
         value={isControlled ? value : savedDefaultValue}
       >
-        {children}
+        {children || <AutoFields components={components} />}
```

Comment:
I wonder if there'd be value in having a `Formatic` export that includes this behavior? Would that make transitions to the new version a little easier? Maybe not if using a code mod—it'd be just as easy to add an `AutoFields` component as the child.

Reply:
Maybe? I didn't specify this, but I'm thinking right now, we push a "clean" version as far as we can go, and then we think about what that means to existing Formatic usage after that. Do we use a codemod or an adapter or just a "good luck"? I'm not sure yet, but I think if we worry too much about the transition now, it might dirty up the rewrite. Or at least make it harder on our brains. 😄